utils/modeler/modeler.go

```diff
@@ -59,9 +68,9 @@ func (m *Modeler) mapToModel(data map[string]string, context string, rv reflect.
 			// We're nested... use some recursion...
```

Comment:
This _might_ be a problem, but since it's not introduced by this PR, I'm going to shamelessly kick that can down the road.